src/sql/parts/notebook/models/notebookModel.ts

```diff
@@ -400,25 +411,84 @@ export class NotebookModel extends Disposable implements INotebookModel {
 				this._activeConnection = undefined;
 			}
 
-			clientSession.initialize();
+			await clientSession.initialize();
 			this._sessionLoadFinished = clientSession.ready.then(async () => {
```

Comment:
Given ready is resolved inside the initialize method, do we need to wait on ready here? Can't we just remove the ready.then code? Thanks!

Reply:
Yes. we can remove it.